docs/rest_api.rst

```diff
@@ -357,7 +357,8 @@ Example Request
         "partner_address": "0x61c808d82a3ac53231750dadc13c777b59310bd9",
```

Comment:
If we do decide to add the `reveal_timeout` to the rest API then we most definitely should document here that it should **not be provided** in most circumstances. It's only an advanced user feature.

Reply:
Added observation in documentation discouraging messing with this option.